### PR TITLE
l10n: if english variant is not specified then choose US as default

### PR DESCRIPTION
**Pull request by lpranam**
Signed-off-by: Pranam Lashkari <lpranam@collabora.com>
Change-Id: Iba4f96c75d69e52044f6460c4e26d0ab1d187d3c

* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: common/Session.cpp [hunk 1]
```diff
@@ -157,7 +157,10 @@ void Session::parseDocOptions(const StringVector& tokens, int& part, std::string
         }
         else if (name == "lang")
         {
-            _lang = value;
+            if (value == "en")
+                _lang == "en-US";
```
Comment (merttumer) [inline common/Session.cpp:161]: LGTM, except this typo :=)

Comment (lpranam) [reply common/Session.cpp]: done. thanks

**Review by timar — APPROVED** [review]
You are right, it's easier to fix it here, than finding out what is wrong with the language fallbacks in core. :)

**Comment by timar** [comment]
We need this in co-6-4, too.

**Pull request merged**